### ISSUE TITLE
When using DNS over TLS, DNS over ProtonVPN stops working

### ISSUE THREAD

**Issue by Anonymous941** [ProtonVPN/proton-vpn-gtk-app#79]
>  We are happy to answer your questions about the code or discuss technical ideas. 
 
Please complete the following checklist (by adding [x]):
- [X] I have searched open and closed issues for duplicates
- [X] This isn't a feature request
- [ ] This is not a report about my app not working as expected

----------------------------------------
I'm using DNS over TLS to circumvent censorship on a particular WiFi network, which blocks ProtonVPN (and because there's no stealth mode, I'm forced to not use the VPN on that network).  However, when I connect to ProtonVPN again, DNS stops working.

The only workaround I found is to manually update `/etc/resolv.conf` every time I enable the VPN and add a custom DNS.  This causes DNS requests to be leaked to that DNS resolver, but I'm still able to access the internet

**Comment by calexandru2018** [comment]
Hey @Anonymous941 , we'll be introducing the ability to add custom DNS to the app very soon. Once you get the update, please let me know if that solves your use case.

**Comment by john482847573** [comment]
Protonvpn still doesn't function when using dns over tls in systemd resolved, even when custom dns is set in the proton vpn app.

**Comment by 0xGingi** [comment]
I'm having this issue also, my router has DoT/DoH support, and DNS does not work on ProtonVPN unless I change the DNS on my machine to unencrpyted/not using DoT or DoH

**Comment by john482847573** [comment]
Is there any update on this?

**Comment by 0xGingi** [comment]
I seem to no longer have the issue

**Comment by Anonymous941** [comment]
> I seem to no longer have the issue

I still do, are you using the systemd resolved DNS (127.0.0.53)?

**Comment by 0xGingi** [comment]
> > I seem to no longer have the issue
> 
> I still do, are you using the systemd resolved DNS (127.0.0.53)?

No, I'm using network manager instead of resolved, I did recently change my router to use DoH instead of DoT, which may? have been the problem but no clue

**Comment by Anonymous941** [comment]
I think the issue is only with resolved, as @Felting pointed out

**Comment by 0xGingi** [comment]
I don't think it was only with resolved, as I was having the issue also

**Comment by Anonymous941** [comment]
This is no longer happening on the latest version, was the issue fixed?

**Comment by john482847573** [comment]
Still doesn't work for me

**Comment by TuxBalazs** [comment]
Still doesn't work. When I set the DNSOverTLS option to yes in /etc/systemd/resolved.conf it stops working. When I disable DoT, it starts working again.
